code/tool/camera-projection/src/cameraprojection.cpp

```diff
@@ -120,8 +122,10 @@ void CameraProjection::setUp()
 {
   odcore::base::KeyValueConfiguration kv = getKeyValueConfiguration();
 
-  m_cameraName = kv.getValue<std::string>("core-tool-camera-projection.cameraname");
-  m_transformationMatrixFileName = m_cameraName + "-pixel2word-matrix.csv";
+  m_cameraName = 
+      kv.getValue<std::string>("core-tool-camera-projection.cameraname");
+  m_debug = (kv.getValue<int32_t>("core-tool-camera-projection.debug") == 0);
+  m_transformationMatrixFileName = "/opt/opendlv.core.configuration/" + m_cameraName + "-pixel2word-matrix.csv";
```

Comment:
You should use the relative path of this module (i.e., "./") and make sure that the corresponding docker-compose.yml file contains the right setting for working directory, i.e. -w